src/__tests__/index.ts

```diff
@@ -200,3 +260,53 @@ describe('verifyPresentation', () => {
     expect(verifyPresentation(BASIC_JWT, resolver)).rejects.toThrow()
```

Comment:
```suggestion
    expect.assertions(1)
    expect(verifyPresentation(BASIC_JWT, resolver)).rejects.toThrow()
```

Reply:
That specific function is actually not async :)

Reply:
There is a `Promise` involved, so it is async, just not using the `async` keyword